assets/js/base/components/cart-checkout/product-sale-badge/index.js

```diff
@@ -17,18 +17,26 @@ import ProductBadge from '../product-badge';
  * @param {Object} props            Incoming props.
  * @param {Object} props.currency   Currency object.
  * @param {number} props.saleAmount Discounted amount.
- *
+ * @param {string} props.format     Format to change the price.
```

Comment:
I think we need to mark it as optional?
```suggestion
 * @param {string} [props.format]     Format to change the price.
```